velox/substrait/SubstraitParser.h

```diff
@@ -75,7 +75,7 @@ class SubstraitParser {
   std::string getFunctionName(const std::string& functionSpec) const;
 
   /// Extracts argument types for a function from specified compound name.
-  void getFunctionTypes(
+  void getSubFunctionTypes(
```

Comment:
No need to add "Sub" as this method is inside SubstraitParser class.

Reply:
OK, it is fixed.